t/lib/OpenQA/SeleniumTest.pm

```diff
@@ -219,12 +219,12 @@ sub javascript_console_has_no_warnings_or_errors {
 
             # ignore error responses in 13-admin.t testing YAML errors
             next if ($msg =~ qr/api\/v1\/exp.*\/job_templates_scheduling\/1003 - Failed.*/);    # uncoverable statement
-        }
-        elsif ($source eq 'javascript') {
+
             # ignore when the proxied ws connection is closed; connection errors are tracked via the devel console
             # anyways and when the test execution is over this kind of error is expected
             next if ($msg =~ qr/ws\-proxy.*Close received/);
-
+        }
+        elsif ($source eq 'javascript') {
```

Comment:
I really wish it wasn't so extremely racy. I now managed to reproduce the message with the `javascript` source. So it seems we need it in both afterall. Even if you can easily never see it in a few hundred runs...